packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.scss

```diff
@@ -15,27 +15,29 @@
 
 .root {
   position: relative;
+  ::slotted(fluent-tree-item) {
+    --tree-item-nested-width: 2em;
+  }
```

Comment:
we should set the --card-height property to auto so that it expands/contacts with the content inside the card